<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processMultireadsView</class>
 <widget class="QWidget" name="processMultireadsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>processMultireadsView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListWidget" name="queueListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Input reads (*.bam file)</string>
          </property>
          <property name="buddy">
           <cstring>inputLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="inputLineEdit"/>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="inputFind">
          <property name="text">
           <string>find</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Output reads (*.bam file)</string>
          </property>
          <property name="buddy">
           <cstring>outputLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="outputLineEdit"/>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="outputFind">
          <property name="text">
           <string>find</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>calculate weights, using allocation distance:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="spinBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputLineEdit</tabstop>
  <tabstop>inputFind</tabstop>
  <tabstop>outputLineEdit</tabstop>
  <tabstop>outputFind</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
